lib/openstack_query/query_blocks/query_output.py

```diff
@@ -302,8 +306,5 @@ def _flatten_list(data: List[Dict]) -> Dict:
         res = {}
         for key in keys:
             vals = [d[key] for d in data]
-            # converting into a set preserving order
-            # https://stackoverflow.com/a/53657523
-            vals = list(dict.fromkeys(vals))
-            res[key] = vals
+            res[key] = list(vals)
```

Comment:
Nit: this can be done as
```
res[key] = [d[key] for d in data]
```

Since you ask for a list then cast to a list again

Reply:
it's already a list because list comprehension above